common/cmake/devices.cmake

```diff
@@ -269,15 +268,10 @@ function(DEFINE_DEVICE_TYPE)
   # If the SCRIPT has dependencies, SCRIPT_DEPS can be used to be passed to the
   # SCRIPT command.
   #
-  # If the UPDATE_TILES option is active, the architecture XML file will go through
-  # an additional step and is updated to have include the `tiles` tags.
-  #
   # DEFINE_DEVICE_TYPE defines a dummy target <arch>_<device_type>_arch that
   # will build the merged architecture file for the device type.
-  #
-  # If UPDATE_TIMINGS is set merged arch.xml file will be processed to inject
-  # timing values using data from prjxray-db/$ARCH/timigs/*sdf files
-  set(options UPDATE_TIMINGS UPDATE_TILES)
+
+  set(options UPDATE_TIMINGS UPDATE_TILES ADD_PACK_PATTERNS)
```

Comment:
Still need to remove the relevant options?

Reply:
Yep, done.